docs/watch_tutorial/index.md

```diff
@@ -0,0 +1,362 @@
+
+```css
+@font-face {
+	font-family: 'digital-7';
+	src: url('./fonts/digital-7\ \(mono\).ttf')
+}
+
+body, html {
+	background-color: transparent;
+}
+
+.Watch {
+	font-family: 'digital-7', monospace;
+	font-size: 2.8rem;
+	color: #4Af626;
+	margin-top: 47%;
+	margin-left:48%;
+	max-width: 1.5rem;
+	text-align: center;
+}
+```
+et voilà!
+
+![hello-aardvark-digital](./media/hello-aardvark-dig.jpg)
+
+
+# Step 3: _Time After Time_ - Adding the Watch model
+
+After scouring Poly and Sketchfab for good, free digital watch models, I found this [great-looking, downloadable photogrammetized model of a Casio G-Shock](https://sketchfab.com/3d-models/casio-g-shock-digital-watch-b6466b64dd8b495c99f24a5054491f4d) by [jeandiz](https://sketchfab.com/jeandiz). Clocking in at ~25MB, its textures are way too nice and detailed for our purposes, so I've resized them in imagemagick and did some model cleanup in Blender; you can find the optimized, <1MB gltf file [here](media/watch.glb). 
+
+![Casio](media/casio.jpg)
+
+Let's add this make this the gadget preview model while we're at it! in `manifest.webmanifest`, under `icons`, change to:
+```webmanifest
+	"icons": [
+		{
+			"src": "models/watch.glb",
+			"type": "model/gltf-binary"
+		}
+	],
+```
+**important** - remember to add the watch (and all gltf models) to your **`dist/models`** folder, not `src/models`! gltf models do not get copied over as part of the compilation flow - the path is relative from `dist/manifest.webmanifest`.
+
+Now that our gadget preview is all set up, let's add the watch to the gadget itself:
+```tsx
+<div>
+	<AvGrabbable dropOnHooks={true}>
+		<AvSphereHandle radius={0.1} />
+		<AvTransform uniformScale={0.2}>
+			<AvPanel interactive={true} >
+				<AvTransform translateX={.063} translateY={-.28} translateZ={-.023}>
+					<AvModel uri="./models/watch.glb" scaleToFit={{ x: 0.35, y: 0.35, z: 0.35 }} />
+				</AvTransform>
+				<div className="Watch">
+				Hello Aardvark!
+				</div>
+			</AvPanel>
+		</AvTransform>
+	</AvGrabbable>
+</div>
+```
+
+Not *too* much happening here. Inside AvPanel, we've added a `<AvTransform>` component and inside *it*, an `<AvModel>` component. We're using the `<AvTransform>` component to place the model where we want it in the gadget and `<AvModel>`'s `scaleToFit` property to lock the model's scale into an (imaginary) box of a certain size. The values are a result of trial and error - our main concern is having the watch in a reasonable place and setting the digits' size and location with CSS.
+
+# Step 4: _Time for Action_ - Making it interactive
+
+Alright, so we've got a watch; time (heh) to make it tick! 
+In the spirit of old school digital watches, let's have a `time` display and - when the screen is "pressed" - a `date` display as well. We'll start by adding a "watchDisplay" string and an instance of a "watchDisplayType" enum to our state interface, to manage the display state:
+
+```tsx
+enum watchDisplayType {
```

Comment:
Opening brace on the new line please

Reply:
newline brace across the file (for consistency), thanks